### DISCUSSION TITLE
[Esercitazione 13 Dicembre 2023] Esercizio 3

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#322] [Esercizi - Viaggi/Piazza]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/8a60a93f-3230-4e15-a984-17aa7829ac80)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/757ee47e-86ad-4825-b31b-d9591ba9e116)

**Reply by Frisbon** [reply]
Perdonami, penso che la riduzione a scala venga diversamente
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/84731341/60a4ca3d-d83a-4410-abcf-6165f661de87)

**Reply by CiottoloMaggico** [reply]
Ciao, ti sei dimenticato un segno sul 4 (seconda colonna 4 riga). Penso sia per quello che viene diverso

**Reply by Frisbon** [reply]
Grazie, rifaccio l'esercizio e vedo se viene.

**Comment by Frisbon** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/84731341/60014cb4-2bdf-4f92-ade3-1f85f3c9dc80)

> se sbaglio ditemelo please
